src/database/query-table.c

```diff
@@ -359,12 +359,13 @@ void DB_read_queries(void)
 			continue;
 		}
 
-		const int status = sqlite3_column_int(stmt, 3);
-		if(status < QUERY_UNKNOWN || status >= QUERY_STATUS_MAX)
+		const int status_int = sqlite3_column_int(stmt, 3);
+		if(status_int < QUERY_UNKNOWN || status_int >= QUERY_STATUS_MAX)
 		{
-			logg("FTL_db warn: STATUS should be within [%i,%i] but is %i", QUERY_UNKNOWN, QUERY_STATUS_MAX-1, status);
+			logg("FTL_db warn: STATUS should be within [%i,%i] but is %i", QUERY_UNKNOWN, QUERY_STATUS_MAX-1, status_int);
 			continue;
 		}
+		const enum query_status status = status_int;
```

Comment:
This change ensures we cannot forget to add new status types in the future (the `switch` statement would complain during compilation, otherwise).